src/app/routes/homePage/getInitialData/index.test.js

```diff
@@ -0,0 +1,130 @@
+import fetchMock from 'fetch-mock';
```

Comment:
We can probably delete most of this 😄 and just check that it returns a status of 200 and an empty `pageData` object

Reply:
Test added!